nafas/functions.py

```diff
@@ -225,9 +253,11 @@ def run(program_data):
         time.sleep(unit / 2)
         for index, item in enumerate(ratio):
             if item != 0:
+                item_name = STEP_MAP[index]
+                play_sound(get_sound_path(SOUND_MAP[item_name]))
```

Comment:
I think we need a general error handler here (for `play_sound` function calling)

Reply:
You mean like [this](https://github.com/sepandhaghighi/nafas/pull/23/commits/e16653f8a414b9f6982a4d9f4b2c69b53a37ca66)?